index.bs

```diff
@@ -297,8 +318,25 @@ spec: HEADER-STRUCTURE; urlPrefix: https://httpwg.org/http-extensions/draft-ietf
       then return true.</li>
       <li>Otherwise, for each <var>item</var> in the <a>allowlist</a>:
         <ol>
-          <li>If <var>item</var> is [=same origin-domain=] with
-          <var>origin</var>, then return true.</li>
+          <li>If <var>item</var> is a tuple of an [=origin=] that's [=same origin-domain=] 
+          with <var>origin</var> and the boolean <code>False</code>, then return true.</li>
+          <li>Else if <var>item</var> is a tuple of a <a>serialized-origin</a> that's not [=same origin-domain=] 
+          with <var>origin</var> and the boolean <code>True</code>, then:</li>
+          <ol>
+            <li>If <var>origin</var> is not a <a>tuple origin</a>, then return false.<li>
+            <li>If <var>origin</var>'s <a>host</a> has a null <a>registrable domain</a>, return false.<li>
```

Comment:
This has to refer to the host of an origin using the `for` attribute.

Reply:
Could you explain how you resolved this, as far as I can tell you reordered the words, but that doesn't help the fact that host points to the wrong thing.

Reply:
I think I had misunderstood, is it correct now?